src/nvim/vvars.lua

```diff
@@ -171,6 +171,7 @@ M.vars = {
                          an aborting condition (e.g. |c_Esc| or
                          |c_CTRL-C| for |CmdlineLeave|).
         chan             |channel-id|
+        info             As returned by |nvim_get_chan_info()|.
```

Comment:
I was wondering about that. So it _is_ OK for different autocmds to use `info`  for different things. 

I'll forget about wondering about `chan_info` instead of `info` ;)